lib/ansible/plugins/callback/junit.py

```diff
@@ -141,6 +148,7 @@ def __init__(self):
         self._include_setup_tasks_in_report = os.getenv('JUNIT_INCLUDE_SETUP_TASKS_IN_REPORT', 'True').lower()
         self._hide_task_arguments = os.getenv('JUNIT_HIDE_TASK_ARGUMENTS', 'False').lower()
         self._test_case_prefix = os.getenv('JUNIT_TEST_CASE_PREFIX', '')
+        self._replace_out_of_tree_path = os.getenv('JUNIT_REPLACE_OUT_OF_TREE_PATH', None)
```

Comment:
Should this be using config on newer releases, or is that problematic for backports? Functionally I suppose it makes no difference at the moment...